workos/webhooks.py

```diff
@@ -1,18 +1,32 @@
-from typing import Protocol
-
+from typing import Optional, Protocol, Union
+from pydantic import TypeAdapter
+from workos.resources.webhooks import Webhook
 from workos.utils.request_helper import RequestHelper
 from workos.utils.validation import WEBHOOKS_MODULE, validate_settings
 import hmac
-import json
 import time
-from collections import OrderedDict
 import hashlib
 
+WebhookPayload = Union[bytes, bytearray]
+WebhookTypeAdapter: TypeAdapter[Webhook] = TypeAdapter(Webhook)
```

Comment:
`Webhook` is a discriminated union, not a model. So we need to create a `TypeAdapter` to validate the incoming payload.